lib/pages/authen/register/widgets/email_form.dart

```diff
@@ -0,0 +1,63 @@
+import 'package:flutter/material.dart';
+import 'package:structure_flutter/bloc/bloc.dart';
+import 'package:structure_flutter/core/resource/icon_style.dart';
+import 'package:structure_flutter/core/resource/text_style.dart';
+import 'package:structure_flutter/di/injection.dart';
+class EmailForm extends StatefulWidget {
+  final RegisterState _state;
+  final TextEditingController _emailController;
+
+  EmailForm(this._state, this._emailController);
+
+  @override
+  _PasswordFormState createState() => _PasswordFormState();
+}
+
+class _PasswordFormState extends State<EmailForm> {
```

Comment:
```suggestion
class _EmailFormState extends State<EmailForm> {
```
State's name should be match with Widget's name (kill)